lib/mumuki/laboratory/mumukit/randomizations/with_randomizations.rb

```diff
@@ -0,0 +1,32 @@
+module WithRandomizations
+  def self.included(base)
+    base.include InstanceMethods
+    base.extend ClassMethods
+  end
+
+  module InstanceMethods
+    def seed
+      @seed || 0
+    end
+
+    def seed_with!(seed)
+      @seed = seed
+    end
+  end
+
+  module ClassMethods
+    def interpolations_on(*selectors, with:)
```

Comment:
I'd rename this to `randomize` since we are using the term `interpolation` with a different meaning: replace and mix sections of the code using `Mumukit::Directives`

Reply:
right! I was planning on somehow splitting the interpolation and randomization logic, but I just couldn't come up with a way so I gave up, I forgot to rename this :stuck_out_tongue:

Reply:
:+1: